ledger/src/blockstore_db.rs

```diff
@@ -339,6 +342,19 @@ pub mod columns {
     /// * value type: [`blockstore_meta::OptimisticSlotMetaVersioned`]
     pub struct OptimisticSlots;
 
+    #[derive(Debug)]
+    /// The merkle root column
+    ///
+    /// This column  stores the merkle root for each FEC set.
+    /// Each merkle shred is part of a merkle tree for
+    /// its FEC set. This column stores that merkle root.
+    ///
+    /// Its index type is (Slot, FEC) set index.
+    ///
+    /// * index type: `crate::shred::ErasureSetId` `(Slot, fec_set_index: u64)`
+    /// * value type: [`solana_sdk::hash::Hash`]`
+    pub struct MerkleRoot;
```

Comment:
@behzadnouri I decided to add a new column instead of modifying the existing `ErasureMeta` because `ErasureMeta` is only populated once we insert a coding shred.

This approach will allow us to submit a duplicate proof if we observe 2 data shreds with differing merkle roots.

Reply:
ah you're right 🤦 i'll stick to the original approach of modifying the erasure meta instead.

Reply:
> If they have different index and different merkle roots, then you can't determine if they belong to the same FEC set or not without receiving a coding shred from the same FEC set

Out of curiosity, why is it not enough to compare `(shred.slot(), shred.fec_set_index())`?

Reply:
Discussed this offline.
@AshwinSekar is right. `(shred.slot(), shred.fec_set_index())` will identify the FEC set.

The 2 cases below are left out but they can be tracked separately.
> * another scenario for duplicate blocks is that you receive a coding shred which indicates data shreds with indices [10..20] belong to the same FEC set with fec_set_index = 10, but then you have a data shred with index 17 (i.e. within that range) but different fec_set_index.
> * Another case is that we have 2 coding shreds which indicate overlapping FEC sets.